frontend/lib/resolution/can-pass.cpp

```diff
@@ -811,6 +812,35 @@ bool CanPassResult::canInstantiateBuiltin(Context* context,
   return false;
 }
 
+
+CanPassResult
+CanPassResult::canInstantiateOwnedSharedHelper(Context* context,
+                                               const types::ClassType* actualCt,
+                                               const types::CompositeType* formalCt) {
```

Comment:
Since the `_owned` record can be instantiated, I think this is insufficient; you will allow `_owned(C)` to be instantiated with `owned`, I think.